src/core.ts

```diff
@@ -79,14 +80,12 @@ export class Core {
   readonly baseOptions: ESLint.Options;
 
   constructor(config: Config) {
-    this.config = config;
-    this.baseOptions = {
-      cache: this.config.cache ?? DEFAULT_BASE_CONFIG.cache,
-      cacheLocation: this.config.cacheLocation ?? DEFAULT_BASE_CONFIG.cacheLocation,
-      rulePaths: this.config.rulePaths,
-      extensions: this.config.extensions,
-      cwd: this.config.cwd,
+    this.config = {
+      ...DEFAULT_BASE_CONFIG,
```

Comment:
This fixes a bug where `DEFAULT_BASE_CONFIG.quiet` and `DEFAULT_BASE_CONFIG.formatterName` were only being used in the CLI, and not the programmatic API. I think my new code implements the original intention of `DEFAULT_BASE_CONFIG`.

Reply:
As you say, this is a bug. Thanks for fixing it.